composer.json

```diff
@@ -14,9 +14,11 @@
   },
   "require": {
     "php": ">=5.5",
-    "doctrine/lexer": "^1.0.1"
+    "doctrine/lexer": "^1.0.1",
+    "symfony/polyfill-intl-idn": "^1.10"
   },
   "require-dev": {
+    "ext-intl": "*",
```

Comment:
Relates to #231 , since intl is still required for the spoof checker.
Maybe you can revert this change to avoid blocking this PR because of the INTL dependency fix.

Reply:
re: `SpoofChecker`, what're your thoughts [regarding forking `SpoofCheckValidation` to it's own package that requires `ext-intl`](https://github.com/egulias/EmailValidator/pull/229#discussion_r362545036) ?

Reply:
Answered. However, we can still remove this changes from the current PR in order to merge it and solve the INTL issue there.

Reply:
removed in affd3dd